.github/workflows/python.yml

```diff
@@ -0,0 +1,27 @@
+# Python unit tests
+name: Python
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2.1.1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip-upgrade
+          pip install -r requirements.txt
+          pip install -r requitements-dev.txt
+      - name: black
```

Comment:
Have you thought to set up pre-commit?  This will make it easier to set up the environment for new contributors. However, we can do it separately. If you are interested, create a ticket and you can call me, I will take care of it.

Reply:
That's awesome, yes I really like pre-commit for this. I'll ping you if needed